pygments/lexers/matlab.py

```diff
@@ -647,6 +647,15 @@ class OctaveLexer(RegexLexer):
         ],
     }
 
+    def analyse_text(text):
+        """Octave is quite hard to spot from a casual glance, but we'll assume
+        that most examples will have an integer vector or matrix somewhere."""
+        # A \ B is actually quite uncommon outside of Matlab/Octave
+        if re.match(r'\w+\s*\\\s*\w+',text):
+            return 0.05
+        if re.match(r'\[\s*(?:(?:\d+\s*)+;\s*)+\s*\]', text):
```

Comment:
This is a quite complicated regex that might backtrack (not catastrophically, but still a bit much to do for analyse_text...)

Reply:
I just tried those out and both aren't really good indicators either. I'm inclined to `return 0` until someone has a better idea.